packages/prism-editor/src/PrismEditor.tsx

```diff
@@ -89,7 +84,7 @@ export const PrismEditor: FC<PrismEditorProps> = ({
         <Pre className={className} style={style}>
           {tokens.map((line, i) => (
             <Line key={i} {...getLineProps({ line, key: i })}>
-              <LineNo>{i + 1}</LineNo>
+              {language === 'json' || <LineNo>{i + 1}</LineNo>}
```

Comment:
I see that `LineNo` uses `span` under the hood. FWIW @looker/components has a `Span` component now, might want to use that instead. Same applies for L99